clients/ptranslator/src/main/java/org/hawkular/metrics/clients/ptrans/PTrans.java

```diff
@@ -60,7 +60,7 @@
  * @author Thomas Segismont
  */
 public class PTrans {
-    private static final Logger LOG = LoggerFactory.getLogger(PTrans.class);
+    private static final PTransLogger log = PTransLogging.getPTransLogger(PTrans.class);
```

Comment:
Just curious: why does this not follow `Logger log = Logger.get...` as in all the other cases? Is that due to limitations in the annotation processor?